test.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
What is this file?

Reply:
ah snap! sorry about that, by accident in here... removed

Reply:
deleted and rebuild now. sorry about that!